README.md

```diff
@@ -329,6 +329,8 @@ expect(double).to receive(:msg).exactly(3).times.and_return(value1, value2, valu
 expect(double).to receive(:msg).and_raise(error)
   # error can be an instantiated object or a class
   # if it is a class, it must be instantiable with no args
+expect(double).to receive(:msg).and_raise(StandardError.new(some_arg))
+  # instantiated error object for class which requires args
```

Comment:
If you see above, you can see we mention this, how do you feel about expanding that instead?
e.g.
```
# `error` can be an instantiated object e.g `StandardError.new(some_arg)` or a class `StandardError`
# If a class is provided it must be instantiable with no arguments
```

Reply:
Sure, that's more concise. Done!